### ISSUE TITLE
Feature request: values in Devices interrupt activity

### ISSUE THREAD

**Issue by Saentist** [mikaku/Monitorix#286]
![int1z 1day](https://user-images.githubusercontent.com/16975883/87425870-0d54cd00-c5e7-11ea-8c9e-7c94003088d9.png)

When there is so many devices is it possible to see values after them, currently I fill as colour blind to recognise device with overload one of cores.

![proc29 1day](https://user-images.githubusercontent.com/16975883/87426369-bbf90d80-c5e7-11ea-9b44-af9259f8a960.png)

**Comment by mikaku** [comment]
Yes, the `int.pm` module needs a complete rewrite, as I stated in #77.
Please, feel free to provide there a complete output (and well formatted) of `cat /proc/interrupts`.

Thanks.

**Comment by Saentist** [comment]
attaching the result
[interrupts.txt](https://github.com/mikaku/Monitorix/files/4934108/interrupts.txt)
